device-types/Cisco/WS-C3560X-48T-L.yaml

```diff
@@ -0,0 +1,118 @@
+---
+manufacturer: Cisco
+model: Catalyst 3560X-48T-L
+slug: cisco-ws-c3560x-48t-l
+part_number: WS-C3560X-48T-L
+is_full_depth: false
```

Comment:
Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

Reply:
Hi! Can you accept this pull request for now? We will see if we can make another pull request adding the data for all in the series since the already existing 3560 also do not have airflow and weight.